registry/warming.go

```diff
@@ -141,9 +141,11 @@ func (w *Warmer) warm(id flux.ImageID, creds Credentials) {
 			// Get the image from the remote
 			img, err := client.Manifest(imageID)
 			if err != nil {
-				if !strings.Contains(err.Error(), context.DeadlineExceeded.Error()) && !strings.Contains(err.Error(), "net/http: request canceled") {
-					w.Logger.Log("err", errors.Wrap(err, "requesting manifests"))
+				if err, ok := errors.Cause(err).(net.Error); ok && err.Timeout() {
```

Comment:
A: Nope -- it's testing for a timeout rather than a deadline. (Also it looks cleaner!)